test/check-machines-disks

```diff
@@ -1140,6 +1140,18 @@ class TestMachinesDisks(VirtualMachinesCase):
             pixel_test_tag='vm-add-disk-modal-custom-path'
         ).execute()
 
+        # check CD/DVD discs correctly assumes raw format for raw file
+        m.execute("qemu-img create -f raw /var/lib/libvirt/images/foobar.raw 1M")
+        self.VMAddDiskDialog(
+            self,
+            device='cdrom',
+            bus_type='scsi',
+            expected_target='sdb',
+            expected_format='raw',
+            file_path='/var/lib/libvirt/images/foobar.raw',
+            mode='custom-path',
+        ).execute()
+
```

Comment:
Should we not also test that a valid cdrom is correctly detected? What happens if we attach an iso now?

```
[jelle@t14s][~/Downloads]%file webui_latest_install.iso
webui_latest_install.iso: ISO 9660 CD-ROM filesystem data (DOS/MBR boot sector) 'Fedora-38-BaseOS-x86_64' (bootable)
[jelle@t14s][~/Downloads]%head --bytes=16 webui_latest_install.iso
3퐐%
```

Or does QEMU expect it to be attached as raw?

Reply:
QEMU handles iso file as raw (which it is).
But adding a test case for iso file too and expecting it to be raw makes sense. 
I updated the PR with a new test